org.uqbar.project.wollok.lib/src/wollok.wlk

```diff
@@ -318,13 +314,16 @@ package lang {
 	 * @since 1.3
 	 */	
 	class Set inherits Collection {
-	
-		override method newInstance() = #{}
+		new(elements ...) {
+			this.addAll(elements)
+		}
+		
+		override method newInstance() = [].asSet()
```

Comment:
I think it is better to directly instantiate the Set. Otherwise we are creating a list instance just to throw-away. In general I don't mind memory-consumption, but in this case this is like a base class for wollok. we can avoid it easily